index.js

```diff
@@ -182,3 +183,45 @@ function LoadProjects() {
 }
 
 document.addEventListener('DOMContentLoaded', LoadProjects, false);
+
+const form = document.getElementById('contactForm');
+const emailFormat = 'Please enter a valid email adress';
+
+function hasValue(input) {
+  if (input === '') {
+    return false;
+  }
+  return true;
+}
+
+function showMessage(input, message) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+}
+
+function validateEmail(input, emailFormat) {
+  const email = input.value.trim();
+  if (!hasValue(input.value)) {
+    return false;
+  }
+  if (email.match(/[A-Z]/g)) {
+    return showMessage(input, emailFormat);
+  }
+  return true;
+}
+
```

Comment:
Kindly make sure to check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: `email === email.toLowerCase()`, 🤔, right now the form is submitting without the right email format.